adapters/sigfox-vertx/src/main/java/org/eclipse/hono/adapter/sigfox/impl/Config.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.sigfox.impl;
+
+import javax.annotation.PostConstruct;
+
+import org.eclipse.hono.adapter.http.HttpProtocolAdapterProperties;
+import org.eclipse.hono.client.RequestResponseClientConfigProperties;
+import org.eclipse.hono.config.ClientConfigProperties;
+import org.eclipse.hono.service.AbstractAdapterConfig;
+import org.eclipse.hono.service.metric.MetricsTags;
+import org.springframework.beans.factory.config.ObjectFactoryCreatingFactoryBean;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.spring.autoconfigure.MeterRegistryCustomizer;
+
+/**
+ * Spring Boot configuration for the Sigfox adapter.
+ */
+@Configuration
+@EnableCaching
+public class Config extends AbstractAdapterConfig {
+
+    private static final String CONTAINER_ID_HONO_SIGFOX_ADAPTER = "Hono Sigfox Adapter";
+    private static final String BEAN_NAME_SIGFOX_PROTOCOL_ADAPTER = "sigfoxProtocolAdapter";
+    private static final String METRICS_PROTOCOL_TAG = "sigfox";
```

Comment:
in analogy to our other adapters, I think that this constant should be defined in `org.eclipse.hono.util.Constants` instead:
```
public static final PROTOCOL_ADAPTER_TYPE_SIGFOX = "hono-sigfox";
```

Reply:
Done.